<launch>	

    <node pkg="robotx_control" type="cmd_vel_repub.py" name="cmd_vel_repub" />
    
</launch>
